docs/source/integrations/casting/roku.md

```diff
@@ -0,0 +1,32 @@
+# Roku
+
+ The Roku OS was purpose-built for streaming and runs across all Roku devices, including streaming players and Roku TVs. On the Roku platform the applications that stream your media are called channels.
+
+## Basic setup
+
+Follow the getting started instructions laid out in the [Roku documentation](https://developer.roku.com/en-gb/overview) to set up your environment. Roku uses it's own scripting language called BrightScript to set up channels/apps.
+
+We recommend using DASH with PlayReady on Roku devices.
+
+the documentation for which can be found in the [content protection](https://developer.roku.com/en-gb/docs/specs/content-protection.md)
+
+```javascript
+customData = "<VUDRM-TOKEN>";
```

Comment:
What do you think about puttin the customData `<VUDRM-TOKEN>` value directly where it is use?